streams/src/main/java/org/apache/kafka/streams/kstream/TimeWindowedCogroupedKStream.java

```diff
@@ -77,7 +77,8 @@
      * {@link StreamsConfig#CACHE_MAX_BYTES_BUFFERING_CONFIG cache size}, and
      * {@link StreamsConfig#COMMIT_INTERVAL_MS_CONFIG commit intervall}.
      * <p>
-     * For failure and recovery the store will be backed by an internal changelog topic that will be created in Kafka.
+     * For failure and recovery the store (which always will be of type {@link TimestampedKeyValueStore}) will be backed by
```

Comment:
For time-windows, it would be a `TimestampedWindowStore` (not tkv-store). (same below for the other method of this class)

Note the signature of `Materialized<K, V, WindowStore<Bytes, byte[]>>` that uses `WindowStore`, not `KeyValueStore`.

Reply:
Updated